docs/source/cdk/aws-setup.rst

```diff
@@ -0,0 +1,33 @@
+****
+aws setup
+****
+**Download/requirements:**
+
+Ensure you have installed nodejs (newer than version 14), AWS CLI, and Docker
+
+- `nodejs <https://nodejs.org/en/download/>`_
+
+- `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_
+
+- `Docker <https://docs.docker.com/get-docker/>`_
+
+**New user**
+
+- log-in to AWS console
+`AWS console <https://aws.amazon.com/console/>`_
+
+- use *imap-sdc-development* for Account ID
+- enter user name and password
+
+- select IAM from *Services* menu or search IAM
+- select *Users* from left menu
+- click your user button
+- select *Security Credentials* tab
+- click *Create Access Key*
+- make note of *Access Key ID* and *Secret Access Key*
+- you can download the .csv file if you want
+- click *Close*
```

Comment:
Nitpick, but can you make this a numbered list?

You can have the numbers automatically generated by using `#.` instead of `-`